util/parsec_file.py

```
@@ -162,52 +161,53 @@ def get_names_of_recursive_functions(self) -> list[str]:
         """
         return [f for f in self.call_graph if self.call_graph.has_edge(f, f)]
 
-    def get_function_names_in_topological_order(self, reverse_order: bool = False) -> list[str]:
-        """Return the function names in this ParsecFile's call graph in topological order.
+    def get_functions_in_topological_order(self, reverse_order: bool = False) -> list[CFunction]:
```

Ruff (0.14.10)

Boolean-typed positional argument in function definition

(FBT001)